### ISSUE TITLE
The code for training PETRV2, the network loss does not drop, has anyone encountered this situation?

### ISSUE THREAD

**Issue by 24werewolf** [megvii-research/PETR#9]
The code for training PETRV2, the network loss does not drop, has anyone encountered this situation?
<img width="1173" alt="屏幕快照 2022-06-27 上午9 49 47" src="https://user-images.githubusercontent.com/20612761/175845752-39d7d752-6a0d-451a-ae70-d1170725a2ed.png">

**Comment by yingfei1016** [comment]
Do you modified the batchsize? Maybe you should adjust the learning rate or use the  GradientCumulativeFp16OptimizerHook（https://github.com/open-mmlab/mmcv/blob/v1.4.0/mmcv/runner/hooks/optimizer.py）

**Comment by Lizhuoling** [comment]
I have met similar phenomenon as you. The loss almost does not drop.

**Comment by Lizhuoling** [comment]
> 

Have you solved the problem?
